tests/spe3/SPE3CASE1.DATA

```diff
@@ -468,7 +468,7 @@ WCONINJE
 
 TSTEP
 -- first year:
-7*52.14285714
+6*52 53
```

Comment:
Is this related to the timezones?

Reply:
The original expression:
```
7*52.14285714
```
correponds to dividing the year in seven identical parts. The problem is that when this is converted to seconds in opm-parser we get:
```
double seconds = 52.14285714 * 86400 = 4505142.8569 
```
and when that is converted to `std::time_t` we loose the fractional part in each conversion, and after seven timesteps have elapsed we are six seconds short of a year. I opted to change this to timesteps which can be perfectly represented as seconds - then we can tests like this:
```
self.assertEqual( datetime.datetime( 2016,1,1), timestep[7])
```
One advantage of this is that we will be future proof if/when opm-parser supports sub-second resolution in the timesteps. Alternatively we can keep things as they are - but then the test must be like:
```
self.assertEqual( datetime.date( 2015, 12, 31, 23, 59, 54), timestep[7])
```
I have a slight preference for the change I have made, but will *not' fight for it.

Reply:
> I have a slight preference for the change I have made, but will not fight for it.

But then it's not SPE3CASE1 anymore, or?

Reply:
This is not imporant for me - I have presented some arguments why I think we should modify the SPE3 test deck, but if the general opinion on the matter is that the SPE3 test deck should be left unmodified I will revert the changes.

> But then it's not SPE3CASE1 anymore, or?

The offical SPEx cases are described in papers from Society of Petroleum Engineers (SPE), there exists no official instances as input decks. The SPE3 case we started with were manually created based on the original papers by a summer student in 2015(?). So - the changes suggested in this PR does not dramatically change the canonical SPE3 state of the case - and anyway that does not really matter that much?

But as I said; this is not important to me. I will not argue the case any further, but happily revert the changes if told to do so.

Reply:
Okay, I agree to the change of SPE3, but I would like a comment on that line
```
TSTEP
-- first year:
6*52 53 -- was originally "7*52.14285714", changed in sunbeam#58
```